src/createNumbersGenerator.js

```diff
@@ -26,7 +26,23 @@
  * @return {number}
  */
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  if (!numbers) {
+    return () => undefined;
+  }
+
+  let i = 0;
+  return () => {
+    while (i < numbers.length) {
+      if (min && max) {
```

Comment:
good, but if we will receive only `max` or `min`?

you can solve this problem with another check or add default values to min and max